php-compatibility.md

```diff
@@ -0,0 +1,19 @@
+# WordPress and PHP compatibility
+
+The relationship between WordPress and PHP is very close, and it is very important that the versions of both systems coincide for proper functioning.
+
+## WordPress and PHP versions matrix
+
+WordPress | PHP min version | PHP max version
```

Comment:
I think it would make sense to either add a column or rename PHP max version to PHP Recommended Version

Reply:
The idea is to add the information about what is the "code PHP version" supported... yes, we have a recommended version (7.1 right now) but this doesn't mean it does not work with 7.4 o 5.6.20... this table means that "if you detect a WP installation with 4.9 version, you, as a hoster, only can update, force, or whatever to "that max version". If you upgrade to a greater version it may break...

Also, recommended PHP versions does not correlate with WP versions. We changed the recommendation a month ago, without any WP launch.